hedera-node/src/main/java/com/hedera/services/legacy/handler/SmartContractRequestHandler.java

```diff
@@ -768,7 +768,7 @@ private long rbhPriceTinyBarsGiven(HederaFunctionality function, SubType subType
 	}
 
 	private long rbhPriceTinyBarsGiven(HederaFunctionality function, Timestamp at) {
-		return rbhPriceTinyBarsGiven(function, at);
+		return rbhPriceTinyBarsGiven(function, SubType.DEFAULT, at);
```

Comment:
This is not necessary change

Reply:
If not included SubType.DEFAULT, the invocation of rbhPriceTinyBarsGiven(HederaFunctionality function, Timestamp at) breaks ServicesMain execution with StackOverflowError since we made an endless recursion without exit